src/main/app/claims/models/claim.ts

```diff
@@ -646,4 +646,8 @@ export class Claim {
   private hasOrderBeenDrawn (): boolean {
     return !!this.directionOrder
   }
+
+  public isIntentionToProceedEligible (): boolean {
+    return this.directionsQuestionnaireDeadline.isAfter(MomentFactory.parse('2019-09-29'))
```

Comment:
2019-09-29??

Reply:
Do you need method to be public?

Reply:
if we need this date for some reason- it may be useful to name it as a constant with some name explaining why we use this date.

Reply:
we set the date of the deadline to be 19 days in the future. so a claim that gets a full defence on monday will have a deadline date set of the 30th